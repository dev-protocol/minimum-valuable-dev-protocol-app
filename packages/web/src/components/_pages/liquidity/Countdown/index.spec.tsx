import React from 'react'
import { render } from '@testing-library/react'
import { Countdown } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/_pages/liquidity/geyser/hooks.ts')
jest.mock('src/fixtures/_pages/liquidity/uniswap-pool/hooks.ts')

describe(`${Countdown.name}`, () => {
  test('Snapshot', () => {
    const component = render(<Countdown endAtSec={1234567}></Countdown>)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
