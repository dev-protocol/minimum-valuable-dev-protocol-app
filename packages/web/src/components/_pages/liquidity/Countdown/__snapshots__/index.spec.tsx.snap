// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Countdown Snapshot 1`] = `
<body>
  <div>
    <div
      class="ant-statistic"
    >
      <div
        class="ant-statistic-title"
      >
        Program duration
      </div>
      <div
        class="ant-statistic-content"
      >
        <span
          class="ant-statistic-content-value"
        >
          0 day(s) 00:00:00 left
        </span>
      </div>
    </div>
  </div>
</body>
`;
